fda/graphics.py

```diff
@@ -0,0 +1,212 @@
+"""Graphics Module to visualize FDataGrid."""
+from mpl_toolkits.mplot3d import Axes3D
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+import math
+from .depth_measures import *
+from .grid import FDataGrid
+
+__author__ = "Amanda Hernando Bernabé"
+__email__ = "amanda.hernando@estudiante.uam.es"
+
+
+def fdboxplot(fdgrid, fig=None, method=modified_band_depth, prob=[0.5], fullout=False, factor=1.5, barcol="blue",
```

Comment:
I thought that we said that we would return a class containing all the information in the boxplot, and which can be plotted, so that retrieving the information is independent on the plot. Why is that not the case, at the end?